### PR TITLE
Fix crash when dragging rail into sky

### LINKED ISSUE #395: Track Editor: Dragging rail into sky crashes game
### Summary
Dragging a rail into the sky crashes the game.

### How to reproduce
1. Open any track in the track editor.
2. Find a rail somewhere.
3. Drag the end of the rail into the sky. You might have to angle the camera a bit to do this.

### Observed behavior
The game crashes with the following error:
`Trying to assign value of type 'Nil' to a variable of type 'Vector3'.` at `res://Editor/Scripts/Editor.gd:73 - at function: handle_drag_mode`.

### Expected behavior
Nothing happens (or the action is canceled), or an error is shown.

### Version
 - Game Version: Commit a5ceb0efdf26b0f0df4005299e31f68ef4b4c307
 - OS: Linux

### Additional information
Video of the bug in action:

https://user-images.githubusercontent.com/22664233/186263389-9c9b2dde-f04a-4c3c-9f5a-c35b83fb4dd0.mp4

### PR DESCRIPTION
**Pull request by HaSa1002**
Fixes #395

### CODE REVIEW

File: src/Editor/Scripts/Editor.gd [hunk 1]
```diff
@@ -480,7 +480,7 @@ var _last_connected_signal: String = ""
 func handle_drag_mode() -> void:
 	var mouse_pos: Vector2 = get_viewport().get_mouse_position()
 	var plane := Plane(Vector3(0,1,0), selected_object.start_pos.y)
-	var mouse_pos_3d: Vector3 = plane.intersects_ray(camera.project_ray_origin(mouse_pos), camera.project_ray_normal(mouse_pos))
+	var mouse_pos_3d := plane.intersects_ray(camera.project_ray_origin(mouse_pos), camera.project_ray_normal(mouse_pos))
```
Comment (DasCapschen) [inline src/Editor/Scripts/Editor.gd:483]: wait, is this seriously the fix!? 😂

Comment (HaSa1002) [reply src/Editor/Scripts/Editor.gd:483]: yeah... The forced type fails because Vector3 can't be nulled (?) and thus we go into nil territory -> Crash

Comment (DasCapschen) [reply src/Editor/Scripts/Editor.gd:483]: oh ffs, wow xD

**Review by DasCapschen — APPROVED** [review]
lgtm

**Pull request merged**